src/Nordea/Components/MultiSelectDropdown.elm

```diff
@@ -60,72 +73,59 @@ init { onFocus, newOutsideClickListener } =
 view attrs dropdown =
     let
-        viewSelectItems index optionGroup =
-            let
-                viewOptionGroup =
-                    (case optionGroup.groupLabel of
-                        Nothing ->
-                            []
-
-                        Just groupLabel ->
-                            [ Html.li
-                                [ css
-                                    [ displayFlex
-                                    , height (rem 1.5)
-                                    , alignItems center
-                                    , justifyContent start
-                                    , marginLeft (rem 0.75)
-                                    , if index == 0 then
-                                        marginTop (rem 0.75)
-
-                                      else
-                                        marginTop (rem 0)
-                                    ]
-                                ]
-                                [ Text.textTinyLight
-                                    |> Text.view [ css [ color Colors.darkGray ] ] [ Html.text groupLabel ]
-                                ]
-                            ]
-                    )
-                        ++ List.map viewOption optionGroup.options
-
-                viewOption option =
-                    Html.li
-                        []
-                        [ Checkbox.init option.name (Html.text option.label) option.onCheck
-                            |> Checkbox.withIsChecked option.isChecked
-                            |> Checkbox.withAppearance Checkbox.Simple
-                            |> Checkbox.view
-                                [ css
-                                    [ width (pct 100)
-                                    , hover [ backgroundColor Colors.coolGray ]
-                                    , padding (rem 0.5)
-                                    ]
-                                ]
-                        ]
-            in
-            Html.ul
+        currentInput =
+            dropdown.inputProperties |> Maybe.map .input |> Maybe.withDefault ""
+
+        viewInput inputProps =
+            input
                 [ css
-                    [ left (rem 0)
-                    , margin (rem 0)
-                    , padding (rem 0)
-                    , listStyle none
-                    , if dropdown.hasFocus then
-                        displayFlex
+                    [ outline none
+                    , important (boxShadow none)
+                    , fontSize (rem 1)
+                    , height (rem 1.75)
+                    , width (pct 100)
+                    , minWidth (pct 70)
+                    , flexGrow (num 1)
+                    , flexBasis (rem 10)
+                    ]
+                , placeholder dropdown.placeholder
+                , value currentInput
+                , onInput inputProps.onInput
+                , Attrs.attribute "role" "combobox"
```

Comment:
this lacks a ref to the uniqueId with the aria-controls attribute

Reply:
done now